packages/scdoc/project.bri

```diff
@@ -53,3 +55,26 @@ export async function test(): Promise<std.Recipe<std.File>> {
 
   return script;
 }
+
+export function liveUpdate(): std.WithRunnable {
+  const src = std.file(std.indoc`
+    let version = http get https://git.sr.ht/~sircmpwn/scdoc/refs
+      | lines
+      | where {|it| $it | str contains "~sircmpwn/scdoc/refs/" }
+      | parse --regex '<a href="/~sircmpwn/scdoc/refs/(?<version>.+)">'
+      | sort-by --natural --reverse version
+      | get 0.version
+
+    $env.project
+      | from json
+      | update version $version
+      | to json
+  `);
+
+  return std.withRunnable(std.directory(), {
+    command: "nu",
+    args: [src],
+    env: { project: JSON.stringify(project) },
+    dependencies: [git, nushell],
```

Comment:
Is git actually needed here? Looks like it's still doing HTML-based matching

Reply:
Indeed, it was a leftover when I was trying to get the git tags from `https://git.sr.ht/~sircmpwn/scdoc`. But it wasn't working as expected, so I decided to switch to the "normal" solution we adopted, aka retrieving the version with a Nushell script.